src/server.cpp

```diff
@@ -3510,10 +3510,15 @@ v3f Server::findSpawnPos()
 	}
 
 	bool is_good = false;
+	// Calculate mapgen edges
+	map.getMapgenParams()->calcMapgenEdges();
```

Comment:
Actually it is, i had to add it because the value was always 31000 (the default).
Remember that `calcMapgenEdges()` is not run until the first SAO item is added to the world.